src/linker/Linker.Dataflow/ValueNode.cs

```diff
@@ -30,6 +30,9 @@ public enum ValueNodeKind
 		MethodParameter,                // symbolic placeholder
 		MethodReturn,                   // symbolic placeholder
 
+		RuntimeMethodHandle,            // known value - MethodRepresented
```

Comment:
Do we need to add these to the various `switch` blocks below?

Reply:
I only found one such block - added it there.